tda/streaming.py

```diff
@@ -189,6 +190,7 @@ async def _init_from_principals(self, principals):
 
         # Initialize miscellaneous parameters
         self._source = principals['streamerInfo']['appId']
+        self._stream_key = principals['streamerSubscriptionKeys']['keys'][0]['key']
```

Comment:
How confident are you that the first key that is returned is always the one to use? I'm fine with using it, but perhaps it would be a good idea to add some `warn`-level logging in case there's more than one?

Reply:
I haven't tested it yet, but I think a reasonable assumption would be that there's a 1-to-1 mapping from the list of accounts to the list of streamerSubscriptionKeys.